tools/cassandra/handler.go

```diff
@@ -193,9 +191,6 @@ func validateCQLClientConfig(config *CQLClientConfig) error {
 	if config.numReplicas == 0 {
 		config.numReplicas = defaultNumReplicas
 	}
-	if config.CQLClient == nil {
-		config.CQLClient = defaultGoCQLClient
```

Comment:
looks like we can remove the file which defines `defaultGoCQLClient`?

Reply:
Yes, I deleted ` tools/cassandra/gocqlClient.go` but Github showns as moving file for some reasons.